lib/optimizely/optimizely_factory.rb

```diff
@@ -57,6 +105,13 @@ def self.custom_instance(
       config_manager = nil,
       notification_center = nil
     )
+      event_processor = BatchEventProcessor.new(
```

Comment:
Can you please add in a doc, default batchsize and flush_interval will be used to setup batchEventProcessor.

Reply:
Done